velox/docs/functions/spark/math.rst

```diff
@@ -48,6 +48,17 @@ Mathematical Functions
 
     Returns the cosecant of ``x``.
 
+.. spark:function:: decimal_round(decimal, scale) -> [decimal]
+
+    The decimal places ``d`` of result depends on precision and scale of ``decimal`` and ``scale`` value.
+    Returns ``decimal`` rounded to ``d`` decimal places using HALF_UP rounding mode. 
```

Comment:
What does 'd' refer to here?

Reply:
It referred to the scale of the result decimal. I revised this part.